app/player.rb

```diff
@@ -36,10 +44,22 @@ def collision_points
     [@top_aft, @bottom_aft, @nose]
   end
 
+  def remove_life
```

Comment:
nit: rename this method `remove_life!` because it changes the state of the object.